nginx.conf

```diff
@@ -220,6 +220,9 @@ server {
   rewrite ^/sdk/python$ /sdk/python/ redirect;
   rewrite ^/cli$ /cli/ redirect;
 
+  # legacy links in some Actor READMEs
+  rewrite ^/scraping/tutorial/introduction$ /academy/apify-scrapers/getting-started permanent;
```

Comment:
Looks good to me.

I am looking to https://github.com/search?q=org%3Aapify+%22%2Fscraping%2Ftutorial%22&type=code

and I see additional links to https://apify.com/docs/scraping/tutorial/web-scraper

Reply:
```suggestion
  rewrite ^/scraping/tutorial/introduction$ /academy/apify-scrapers/getting-started permanent;
  rewrite ^/scraping/tutorial/web-scraper$ /academy/apify-scrapers/web-scraper permanent;
```